neuralmonkey/nn/projection.py

```diff
@@ -74,14 +75,15 @@ def maxout(inputs, size, scope="MaxoutProjection"):
 
 
 def multilayer_projection(input_, layer_sizes, activation=tf.nn.relu,
-                          dropout_plc=None, scope="mlp"):
+                          dropout_keep_prob=None, train_mode: tf.Tensor=None,
```

Comment:
Zase, train_mode sice tady může bejt `None`, ale nepsal bych to takhle. Dal bych `dropout_keep_prob` defaultně na `1.0`, tak jak je to i jinde a train_mode jako povinný parametr, a pak pryč tu podmínku na řádku 86